console/billing/how-to/add-payment-method.mdx

```diff
@@ -22,6 +22,10 @@ Before you can order Scaleway resources, you must add a payment method to your a
 
 ## How to add a credit card
```

Comment:
You can maybe say "How to add a payment card" then no need for the message box

Reply:
I think we need to say debit at some point, bc there was a question about this. however debit is not in the console, so I prefer to keep the box. thanks :D